test/libsolidity/semanticTests/constructor/arrays_in_constructors.sol

```diff
@@ -26,6 +26,6 @@ contract Creator {
 // compileViaYul: also
 // ----
 // f(uint256,address[]): 7, 0x40, 10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 -> 7, 8
-// gas irOptimized: 472538
+// gas irOptimized: 478315
```

Comment:
The difference is evaluating `keccak` inside a constructor. Even though currently, `keccak` doesn't get evaluated for deploy code, this example seems to be an edge case. Here is the diff: https://gist.github.com/hrkrshnn/61eacc72681811ab5d3d992b2fcab8ca#file-keccak-diff

Let me see if this can be fixed.

Reply:
I made an issue about what is causing this: https://github.com/ethereum/solidity/issues/11118. Would be rather difficult to improve this now.